src/redisearch_rs/ffi/build.rs

```diff
@@ -33,6 +33,7 @@ fn main() {
     let deps = root.join("deps");
     let bindings = bindgen::Builder::default()
```

Comment:
maybe we could create a list of binding interfaces outside for better readability?

Reply:
I mean have a:

`let include_api_headers = ['....']`